contracts/oracle/ChainlinkOracleWrapper.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.6.0;
```

Comment:
Don't worry about updating the solidity version for now, we'll update the codebase to 0.8.4 in the next release per issue 4.9

Reply:
ok